auto3dseg/algorithm_templates/dints/scripts/search.py

```diff
@@ -167,9 +172,9 @@ def run(config_file: Optional[Union[str, Sequence[str]]] = None, **override):
         )[dist.get_rank()]
     logger.debug(f"val_files:, {len(val_files)}")
 
-    train_cache_rate = float(parser.get_parsed_content("searching#train_cache_rate"))
+    train_cache_rate = float(parser.get_parsed_content("train_cache_rate"))
```

Comment:
Does this need to be `searching#train_cache_rate`?

Reply:
I have fixed this line.